pages/linux/wg.md

```diff
@@ -0,0 +1,12 @@
+# wg
+
+> Manage the configuration of WireGuard interfaces.
+> More information: <https://www.wireguard.com/quickstart/>.
+
+- Check status of currently active interfaces:
+
+`sudo wg`
+
+- Print out a new private and public key pair:
+
+`wg genkey | tee /dev/tty | wg pubkey`
```

Comment:
Does `wg pubkey` produce it's own output?

Reply:
Yes. I assume this is related to contribution guideline 4. I'll make `genkey` and `pubkey` their own entries.

Reply:
Okay.  I was just wondering: if `genkey` and `pubkey` both produce their own output, I don't think it's a good idea to concatenate both of their outputs.

Reply:
The official quickstart guide recommends doing so. `wg genkey | tee privatekey | wg pubkey > publickey`
I personally just like doing them both on the command line to get them side-by-side. Should I use the official instructions, keep what I've added or remove the line completely?

Reply:
Ah I see. What do you think about:
- using `wg genkey | tee privatekey | wg pubkey > publickey`, and
- using something like this as the example description: `Generate a public and private key`

How about that?

Reply:
Official instructions it is